code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -982,6 +982,14 @@
 	icon_state = "skrell_hair_f"
 	gender = FEMALE
 
+/datum/sprite_accessory/hair/skrell/skr_tentacleslong
+	name = "Long Skrell tentacles"

```

Comment:
I think the "tentacles" on these two should be capitalized?

Reply:
For consistency it should be. Look at all the other usages of tentacles in that file